generators/app/templates/src/app/components/InputLabel/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function InputLabel({
+  containerClassName,
+  textClassName,
+  dataFor,
+  label,
+  inputClassName,
+  inputName,
+  placeholder,
+  inputId,
+  inputType,
+  handleChange,
+  disabled
+}) {
+  return (
+    <div className={containerClassName}>
+      <span className={`${textClassName} m-bottom-1`} data-for={dataFor}>
```

Comment:
this should be a `<label />` instead of a `<span />`, and also add the htmlFor: in the label